src/ipfs/util.ts

```diff
@@ -9,7 +10,9 @@ type RawDAGLink = {
 }
 
 const rawToDAGLink = (raw: RawDAGLink): DAGLink => {
-  return new dagPb.DAGLink(raw.Name, raw.Tsize, raw.Hash)
+  const h = raw.Hash
+  const c = new OldCID(h.version, h.code, h.multihash.bytes)
+  return new dagPb.DAGLink(raw.Name, raw.Tsize, c)
```

Comment:
This is a bit dumb, but I didn't want to put too much work in here, since this is getting replaced soon anyway by WNFS v2.